pycaret/internal/pycaret_experiment/supervised_experiment.py

```diff
@@ -5029,15 +5029,17 @@ def replace_labels_in_column(pipeline, labels: pd.Series) -> pd.Series:
             df_score = df_score.round(round)
             display.display(df_score.style.format(precision=round))
 
-        label = pd.DataFrame(pred, columns=["Label"], index=X_test_.index)
+        label = pd.DataFrame(pred, columns=["prediction_label"], index=X_test_.index)
```

Comment:
Actually, could we make a switch to a constant defined somewhere instead of hardcoding strings here and in other places? That would make it more maintainable

Reply:
would utils be the place for it or a new file called constants? not sure what best practices dictate here...

Reply:
Let's make `pycaret/constants.py`

Reply:
Can you add constants.py under utils submodule to keep top level relatively clean?

Reply:
That would work, good idea!